data-prepper-plugins/http-source/src/main/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSourceConfig.java

```diff
@@ -13,13 +13,21 @@
 
 import com.amazon.dataprepper.model.configuration.PluginSetting;
 import com.google.common.base.Preconditions;
+import io.micrometer.core.instrument.util.StringUtils;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
 
 public class HTTPSourceConfig {
     static final String PORT = "port";
     static final String REQUEST_TIMEOUT = "request_timeout";
     static final String THREAD_COUNT = "thread_count";
     static final String MAX_CONNECTION_COUNT = "max_connection_count";
     static final String MAX_PENDING_REQUESTS = "max_pending_requests";
+    static final String SSL = "ssl";
+    static final String SSL_CERT = "ssl_certificate";
```

Comment:
We should make our names clearer.
* `ssl_certificate` -> `ssl_certificate_file`
* `ssl_key` -> `ssl_key_file`
* `SSL_CERT` -> `SSL_CERTIFICATE_FILE`
* `SSL_KEY` -> `SSL_KEY_FILE`